consensus/src/test_utils/mock_storage.rs

```diff
@@ -136,15 +140,32 @@ impl MockStorage {
     }
 
     pub fn verify_consistency(&self) -> Result<()> {
+        let order_vote_enabled = false;
+        let window_size = DEFAULT_ENABLED_WINDOW_SIZE.and_then(|window_size| {
+            if window_size <= usize::MAX as u64 {
```

Comment:
Do we have to add this here? Seems it's just a test anyway

Reply:
Was just trying to be safe, but will convert to `u64`